jdi-light/src/main/java/com/epam/jdi/light/elements/composite/Form.java

```diff
@@ -43,20 +43,23 @@
  */
 
 public class Form<T> extends Section {
+    Field setField = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4499053)

Reply:
It was a local variable. Wasn't it?

Reply:
@RomE-HK , for some reason Codacy app is writing the messages from my account name. I've reinstalled integration with Codacy for this repository, so hopefully the next time you will see the message from `codacy-bot`

Reply:
Thank for clarifying!
However my question was addressed to @vladb00mer and his changes in code =)

Reply:
sorry guys I miss notifications( yes all of them are just at the wrong place

Reply:
not local,  just simple variables not at the top of the class